client/components/navigation-header/navigation-header.scss

```diff
@@ -66,7 +66,7 @@
 		color: var(--wp-components-color-gray-900, $gray-900);
 		display: flex;
 		align-items: center;
-		gap: 10px;
+		gap: 12px;
```

Comment:
Align the gap with design:

<img width="220" alt="image" src="https://github.com/user-attachments/assets/cf6ad923-f2c9-4e7a-955f-3e5ba8fecba8" />